lib/components/user/saved-trip-list.js

```diff
@@ -12,95 +12,119 @@ import TripSummaryPane from './trip-summary-pane'
-      content = (
-        <>
-          {accountLink}
-          <h1>You have no saved trips</h1>
-          <p>Perform a trip search from the map first.</p>
-        </>
-      )
-    } else {
-      // Stack the saved trip summaries. When the user clicks on one, they can edit that trip.
-      content = (
-        <>
-          {accountLink}
-          <h1>My saved trips</h1>
-          <p>Click on a saved trip below to modify it.</p>
-
-          {trips.map((trip, index) => (
-            <Panel key={index}>
-              <Panel.Heading>
-                <Panel.Title componentClass='h3'>{trip.tripName}</Panel.Title>
-              </Panel.Heading>
-              <Panel.Body>
-                <TripSummaryPane monitoredTrip={trip} />
-                <ButtonGroup>
-                  <Button bsSize='small' onClick={this._handlePauseOrResumeMonitoring(trip)}>
-                    {trip.isActive ? 'Pause' : 'Monitor' /* TODO: Find better word for 'Monitor' */ }
-                  </Button>
-                  <Button bsSize='small' onClick={this._handleEditTrip(trip)}>Edit</Button>
-                  <Button bsSize='small' onClick={this._handleDeleteTrip(trip)}>Delete</Button>
-                </ButtonGroup>
-              </Panel.Body>
-            </Panel>
-          ))}
-        </>
-      )
-    }
-
+    const { trip } = this.props
     return (
-      <div className='otp'>
-        {/* TODO: Do mobile view. */}
-        <DesktopNav />
-        <div className='container'>
-          {content}
-        </div>
-      </div>
+      <Panel>
+        <Panel.Heading>
+          <Panel.Title componentClass='h3'>{trip.tripName}</Panel.Title>
+        </Panel.Heading>
+        <Panel.Body>
+          <TripSummaryPane monitoredTrip={trip} />
+          <ButtonGroup>
+            <Button bsSize='small' onClick={this._handlePauseOrResumeMonitoring}>
+              {trip.isActive
+                ? <><Glyphicon glyph='pause' /> Pause notifications</>
+                : <><Glyphicon glyph='play' /> Resume notifications</>
+              }
+            </Button>
+            <Button bsSize='small' onClick={this._handleEditTrip}>
+              <Glyphicon glyph='pencil' /> Edit
+            </Button>
+            <Button bsSize='small' onClick={this._handleDeleteTrip}>
+              <Glyphicon glyph='trash' /> Delete
+            </Button>
+          </ButtonGroup>
+        </Panel.Body>
+      </Panel>
     )
   }
 }
 
+// connect to the redux store
+const itemMapStateToProps = () => {}
+
+const itemMapDispatchToProps = {
+  createOrUpdateUserMonitoredTrip: userActions.createOrUpdateUserMonitoredTrip,
+  deleteUserMonitoredTrip: userActions.deleteUserMonitoredTrip,
+  routeTo: uiActions.routeTo
+}
+
+const ConnectedTripListItem = connect(itemMapStateToProps, itemMapDispatchToProps)(TripListItem)
+
+/**
+ * This component displays the list of saved trips for the logged-in user.
+ */
+const SavedTripList = ({ trips }) => {
```

Comment:
I think the code is more readable with the component for which the file is named is at the top of the file.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/231/commits/43d545a1dda5f9308667004d7562119533a3229c